files/zh-cn/web/javascript/reference/global_objects/array/with/index.md

```diff
@@ -93,7 +95,9 @@ console.log(Array.prototype.with.call(arrayLike, 0, 1));
 ## 参见
```

Comment:
Could you please sync the content of 'Calling with() on non-array objects'? The description and example has changed in newest document.

Reply:
✅